Components/Bites/src/OgreWindowEventUtilities.cpp

```diff
@@ -37,7 +37,9 @@ THE SOFTWARE.
 #  endif
 #  include <windows.h>
 #elif OGRE_PLATFORM == OGRE_PLATFORM_LINUX
```

Comment:
```suggestion
#elif OGRE_PLATFORM == OGRE_PLATFORM_LINUX && !defined(OGRE_WAYLAND)
```
as we are not going to write a wayland alternative for this

Reply:
Done